### ISSUE TITLE
Add and Edit Not Working On Dual Edit Panels 

### ISSUE THREAD

**Issue by singhik** [NCIEVS/nci-protege5#563]
@bdionne @fragosog : 

**Steps to reproduce  :** 

Create a concept 
![image](https://user-images.githubusercontent.com/36314371/210012878-e3f1c752-8be7-4b99-a2bd-3a317d8aebf3.png)

Navigate to Dual Edit Panel and perform any operation f.e. Copy  

![image](https://user-images.githubusercontent.com/36314371/210012937-537fe202-9748-453d-858e-be51a8efc91d.png)

Try to Add or Edit any of the Complex properties using + or / buttons in the Pane

![image](https://user-images.githubusercontent.com/36314371/210013034-972063f3-9353-4d77-9c3e-21a2e0be6e71.png)

Edit Fails and does not edit the concept : 

![image](https://user-images.githubusercontent.com/36314371/210013093-3157957b-91cd-43fe-97d0-390698bc4ccb.png)

Try to Add a Complex Property : 

![image](https://user-images.githubusercontent.com/36314371/210013158-6486a6fc-086d-44c1-abf0-0b3bcfc45266.png)

Add also fails and does not add the pty to the concept : 

![image](https://user-images.githubusercontent.com/36314371/210013221-12116713-06db-4273-8832-ddb1812e4571.png)

Delete works sometimes and sometimes does not

**Comment by bdionne** [comment]
This needs to be triaged better. I tried the obvious scenario based on these screenshots and I can't reproduce this. 

I reviewed the codes and commits for this release and see nothing related, perhaps this has been here a while.

Was there a log for this with any exceptions in it?

There are several ways to do this operation, drag/drop, menu selection. Could we have a more detailed enumeration of the steps.

Does this reproduce readily? If possible maybe you can demo it at our weekly status.

I also tried loading the full thesaurus and have no problem with that either.

**Comment by singhik** [comment]
@bdionne  :  I used drag and drop to input the concepts in the dual edit panel and then used Add,Edit,Delete buttons for Complex properties (as in the screenprints above) .

Yes, it is a project that uses \ in the namespace .

I used Manager Role .

Log file attached .
[Uploading Logs.txt…]()

**Comment by bdionne** [comment]
> Yes, it is a project that uses \ in the namespace .

I see, this is likely the culprit, it's new

> Log file attached .

@singhik there's no file attached here, bad link?

**Comment by singhik** [comment]
@bdionne  :  Pfa, log files, 
[Logs.txt](https://github.com/NCIEVS/nci-protege5/files/10354230/Logs.txt)

**Comment by fragosog** [comment]
The "/" project was built with "thesaurus-trimmed-45K-slash.owl.zip", in the nci-protege5/data directory.

**Comment by bdionne** [comment]
I see the issue, it shows up in the log. 

The protege client is configured to use `#` instead of `/` so when you create new classes the `#` is used. This results in a mix of both in the same ontology.

If you change to `/` and restart the problem goes away.

**Comment by bdionne** [comment]
Per our conversation yesterday, QA to verify this is a configuration issue

**Comment by singhik** [comment]
@bdionne : Hi Bob, It's still failing and have tried several iterations , below are the steps with screenprints  : 

Preferences ,showing the Followed by setting : 
![image](https://user-images.githubusercontent.com/36314371/211972141-aafaad78-27c3-4320-8a71-0a0f6564836f.png)

Created two concepts 'Test1' and 'Test2'. doing Dual Edits : 

![image](https://user-images.githubusercontent.com/36314371/211972274-ae69d4f8-50ee-4488-b8b6-59e8d4564bed.png)


Trying to Edit or Add Complex properties using buttons , but unable to perform, only Delete option works : 

![image](https://user-images.githubusercontent.com/36314371/211972408-60982680-29ab-436a-87d6-4146015133d2.png)

![image](https://user-images.githubusercontent.com/36314371/211972489-31946a9c-beda-4adc-af4d-e25ac953d520.png)

![image](https://user-images.githubusercontent.com/36314371/211972546-92b3460f-7325-4616-9bf3-7ccb82678718.png)

![image](https://user-images.githubusercontent.com/36314371/211972600-7012a15d-a313-4aca-8264-77c1c201059f.png)



Logs are attached for reference : 

[Logs.txt](https://github.com/NCIEVS/nci-protege5/files/10398067/Logs.txt)

**Comment by bdionne** [comment]
@singhik 

I believe the issue is that you have the # and / mixed up still. Are you still using the test classes you created before? Is this still the project that uses / ?

I'm pretty sure this is the issue because I can see the exact same exceptions locally if I confuse those IRIs. In order to further triage this I will need the following:

 * For the two classes you are editing in Dual Edits, the full IRI. You can obtain this in the Refactor/Rename menu item.
 * For the parent class you created these under, also the full IRI
 * Do a File/Save and persist the ontology to an OWL file, and push somewhere I can reach or attach it here
 * The project entry in the server config file, you are using for these tests.
 
With this info I can look into this further. This issue is really part of #490

Thank you.


@fragosog We should discuss #490 further. If we're going to allow the changing of IRIs in the preferences maybe we should evaluate what it takes to allow mixed usage in the same ontology. As you can see, it's already causing lots of confusion for users, and once the content is messed up it continues to be a problem.

**Comment by singhik** [comment]
@bdionne : 

IRIs of the two edit classes (Created them afresh, not from before in multiple iterations): 

http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C1000012  & http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C1000013

IRI of the Parent : 

http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl/C36716

Project is located here (with /)  :
https://github.com/NCIEVS/nci-protege5/blob/master/data/thesaurus-trimmed-45K-slash.owl.zip

**Comment by singhik** [comment]
@bdionne  : Yes, did multiple restarts, cleaned fileprefs, did preference settings and restarted , is there any other setting that needs to be changed ?  besides : 

![image](https://user-images.githubusercontent.com/36314371/212132754-ddb442c4-b347-4d1d-9d0d-0d2ea772c695.png)
